build/bootstrap/bootstrap-debian.sh

```diff
@@ -72,3 +72,15 @@ sudo mv /tmp/bazelisk /usr/bin/bazel
 
 # Install the Unison file-syncer.
 . bootstrap/bootstrap-unison.sh
+
+# Configure environment variables for CockroachDB and Managed Service.
+echo 'export PATH="${PATH}:${HOME}/go/src/github.com/cockroachdb/cockroach/bin:/usr/local/go/bin"' >> ~/.bashrc_bootstrap
+echo 'export PATH="${PATH}:${HOME}/go/src/github.com/cockroachlabs/managed-service/bin"' >> ~/.bashrc_bootstrap
+echo -e '\n' >> ~/.bashrc
```

Comment:
I saw you changed the other instance of this line to `echo >> ~/.bashrc`. Just wanted to note this one in case you intended to keep them in sync.

Reply:
Sorry. Missed this!